### ISSUE TITLE
SOI Cscripts Update

### ISSUE THREAD

**Issue by longlostblake** [7Cav/7thCavalry_Training_Missions#30]
Currently we're running an extremely outdated cscripts. Our current cscripts version does not seem to have training classes so they will need making.

Essensial:
- [x] Burn old cScripts version
- [x] Create 'unarmed' training class & loadout for BLN, BRCC.
- [x] Create Instructor class & loadout
- [x] Create Asst. Instructor class & loadout

Non-essensial (Can use standard company specific loadouts)
- [x] Create 'Armed' training class & loadout for BIT, AIT, BLC, ALC (Use standard Charlie company loadouts?)
- [x] Create/Copy 'Bravo Co' training class & loadout (Maybe not needed but will be included)
- [x] Create/Copy 'Alpha Co' training class & loadout (Maybe not needed)
- [x] Create Cadre class & loadout (C.Co FTL loadout?)

Then:
- [x] Apply to SOI BRCC/BLN Map
- [x] Apply to SOI BIT/AIT/BLC/ALC

**Comment by longlostblake** [comment]
Version created with new class for SOI classes, may need more adding for TAS.
Version works on BRCC/BLN but has two errors so i am attaining assistance.
Version should work with BIT/AIT/etc. but will need further testing.

**Comment by longlostblake** [comment]
With a little guidance from Brostrom, sucessfully applied version to BLN/BRCC & BIT.
Also created custom composition for player controlled units which should be universal (needs more testing)

**Comment by longlostblake** [comment]
Current SOI Class:
They are more or less copies of one another for the purpose of adding multiple classes to a map.

```
// Instructors
class rhsusf_usmc_marpat_wd_officer : CAV_SOI_INSTR {};
class rhsusf_usmc_marpat_wd_machinegunner_ass : CAV_SOI_C2_INSTR {};
class rhsusf_usmc_marpat_wd_squadleader : CAV_SOI_C1_INSTR {};
class rhsusf_usmc_marpat_wd_gunner : CAV_SOI_BAS_INSTR {};

// Students

class rhsusf_usmc_marpat_wd_rifleman_m4 : CAV_SOI_STUD {};
class rhsusf_usmc_marpat_wd_rifleman_light : CAV_SOI_C1_STUD {};
class rhsusf_usmc_marpat_wd_rifleman_law : CAV_SOI_C2_STUD {};
class rhsusf_usmc_marpat_wd_riflemanat : CAV_SOI_BAS_STUD {};

**Comment by longlostblake** [comment]
Last set of edits to make.

MIsc bugfixes:
- [x] Create 'training starter crate' sqf for ease.
- [x] Add 'heal' code block to loadouts_training.hpp [Unable to do, left as SQF]
- [x] Remove 240b from starter crate
- [x] clean up S1 office
- [x] Remove OIC junk
- [x] remove garage non-essensials
- [x] remove simpleobject modifier from all HQ (unless required)

Medical:
- [x] Enable Damage to students & Cadre.
- [x] Change CBA settings to enable god*** mode.

Marker Script:
- [x] Redefine marker owners

Armoury Guy:
- [x] Disable interractions (see tact 2 script) or change to object selection.
- [x] Replace with box for ease.

Test:
- [x] cfgloadouts working for each class
- [x] configure new class loadouts (if needed)
- [x] check fireteams have correct group

Compisition
- [x] Recreate clean version of compistion

**Comment by longlostblake** [comment]
Changed `initPlayerLocal.sqf` to include new class types

**Comment by longlostblake** [comment]
Created custom starter crate sqf

Todo: 
- [x] add guer/west to initplayer
- [x] copy marker system from TAS
- [x] add in extra variables/sqms for SOI
- [x] place down template markers and add to composition
- [x] check BLC & ALC test Marker
- [x] Verify Zues Modules are connected

**Comment by longlostblake** [comment]
Added slideshow area, will need at least 1 backup classroom.

**Comment by longlostblake** [comment]
applied ACE self interract 'Heal Target' script to each instructor so they can heal students without being in zues.

**Comment by longlostblake** [comment]
Fixed with commit: 08c51b6
